tests/testthat/helper.R

```diff
@@ -0,0 +1,15 @@
+library("cli")
```

Comment:
this should go away, I think it is needed at the moment because of renv's changing `.libPaths()` whilst testing (and not setting it back).